### DISCUSSION TITLE
Spacing below paragraph

### DISCUSSION THREAD

**Discussion by dkolkin** [quarto-dev/quarto-cli#3589] [Q&A]
Hey there! 
I've started using quarto a few days ago and stumbled upon a rather simple task that I haven't been able to find solution myself. I have none of the experience working with LaTeX, so my apologies for, perhaps, a simple question. How one would set what is called in LibreOffice as "Spacing below paragraph'"?

Thanks a lot!

**Comment by mcanouil** [comment]
Hi,

We need a bit more context. You are talking about LaTeX, but are your using the `pdf` format?
For `html`, this be be achieve with css, _e.g._, for 1.5 lines below:

```css
p {
  padding-bottom: 1.5em;
}
```

PS: could you change the category from "General" to "Q&A"? (via the desktop view of the website)
